packages/astro/src/@types/astro.ts

```diff
@@ -822,6 +822,39 @@ export interface AstroUserConfig {
 		 * ```
 		 */
 		inlineStylesheets?: 'always' | 'auto' | 'never';
+
+		/**
+		 * @docs
+		 * @name build.split
+		 * @type {boolean}
+		 * @default {false}
+		 * @version 2.7.0
+		 * @description
+		 * Defines how the SSR code should be bundled.
+		 *
+		 * When `split` is `true`, Astro will emit a file for each page.
+		 * Each file emitted will render only one page. The pages will be emitted
+		 * inside a `dist/pages/` directory, and the emitted files will keep the same file paths
+		 * of the `src/pages` directory.
```

Comment:
```suggestion
		 * inside a `dist/pages/` directory, mirroring the same file paths as the `src/pages/` directory.
```
I think this is tighter!

Reply:
Same suggestion as above, I think you could say "mirroing the directory structure" if you wanted to!

Reply:
On closer inspection, I think this one is fine as is! Suggested a change only above!